meta-xt-driver-domain/recipes-extended/camerabe/files/camera_be.cfg

```diff
@@ -15,13 +15,13 @@
 // Please see https://elinux.org/R-Car/Tests:rcar-vin for details.
 
 // HDMI_IN -> VIN0 pipeline
-mediactl:
-{
-    link1 = "'rcar_csi2 feaa0000.csi2':1 -> 'VIN0 output':0 [1]";
-    link2 = "'adv748x 4-0070 hdmi':1 -> 'adv748x 4-0070 txa':0 [1]";
-    source_fmt = "'adv748x 4-0070 hdmi':1 [fmt:RGB888_1X24/1920x1080 field:none]";
-    sink_fmt = "'rcar_csi2 feaa0000.csi2':1 [fmt:RGB888_1X24/1920x1080 field:none]";
-}
+// mediactl:
```

Comment:
Do you really need to comment out code? Just delete it.

Reply:
These was the similar question for other meta-layer. I will copy my answer here as is:

"The reason why I commented the code instead of removing it is to have all possible examples visible to the user. As it was discussed internally I will add "This is example for ХХХ use-case" above each commented block."

I am planing to do the same here if no objections.

Reply:
"+"